examples/src/createAgent/middleware/hitl.ts

```diff
@@ -1,5 +1,5 @@
 import { createAgent, HumanMessage, tool } from "langchain";
-import { humanInTheLoopMiddleware } from "langchain/middleware";
+import { humanInTheLoop } from "langchain/middleware";
```

Comment:
We should decide whether we keep the suffix everywhere or drop it.

I suspect some middleware will not be implemented in langchain/middleware but in integration packages. In this case, maybe the the Middleware suffix makes sense so it's easy to recognize things as agent middleware?